todo_app/app.py

```diff
@@ -1,40 +1,51 @@
 from flask import Flask, render_template, request
 
-from todo_app.flask_config import Config
-import requests 
+from todo_app.flask_config import Config # when done with new def can be removed? 
+import requests  
 import os
 from todo_app.data.item import Item
 import todo_app.data.trello_items as trello_items
+from todo_app.data.view_model import ViewModel
 
-app = Flask(__name__)
-app.config.from_object(Config())
-
-
-@app.route('/')
-def index():
-    response_json = trello_items.get_trello_cards()
-
-    items = []
+#new
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+    
+    @app.route('/')
+    def index():
+        response_json = trello_items.get_trello_cards()
 
-    for trello_list in response_json:
-        for card in trello_list['cards']:
-            item = Item.from_trello_card(card, trello_list)
-            items.append(item)
+        items = []
 
-    return render_template('index.html', items = items)
+        item_view_model = ViewModel(items)
 
+        for trello_list in response_json:
+            for card in trello_list['cards']:
+                item = Item.from_trello_card(card, trello_list)
+                items.append(item)
```

Comment:
Although this works, it's quite an odd way of structuring this; first you pass an _empty_ array to the ViewModel, then you populate the array. The fact is that this works because ViewModel is still pointing at the same array you modify, but that's not obvious so we shouldn't rely on it! Switching the order of these two sections would likely make the behaviour more obvious
```suggestion
        for trello_list in response_json:
            for card in trello_list['cards']:
                item = Item.from_trello_card(card, trello_list)
                items.append(item)

        item_view_model = ViewModel(items)
```